catkin_home/src/action_selectors/scripts/parser.py

```diff
@@ -78,6 +87,7 @@ def callback(msg):
     action_code.args = args
     rospy.loginfo(action_code)
     pub.publish(action_code)
+    
 
 def listener():
```

Comment:
This could be called `main()` or something else.

Reply:
you mean def listener() change it to main() ?

Reply:
Yes